exporter/opentelemetry-exporter-otlp/setup.cfg

```diff
@@ -39,3 +39,7 @@ python_requires = >=3.6
 packages=find_namespace:
 install_requires =
     opentelemetry-exporter-otlp-proto-grpc == 1.5.0
+
+[options.entry_points]
+opentelemetry_exporter =
+    otlp = opentelemetry.exporter.otlp.proto.grpc.trace_exporter:OTLPSpanExporter
```

Comment:
OTLP says _there is no specified default as of 1.0_ to not introduce breaking changes. It is left each sig to make the default in this case. What do you think should be default grpc or http/proto?

Reply:
It's unfortunate that the spec doesn't recommend a default. It could lead to different languages having different defaults which can be annoying for new users. I'm leaning towards grpc but I'd prefer to do whatever majority of other SIGs are doing for consistency.

Reply:
I agree. grpc appears to be the mostly supported and preferred.

Reply:
Should this be `otlp_trace`?

Reply:
Good point. We are not allowed to give duplicate entry points and it will be problem when we have metrics and logs. Are we going to document this as special case as opposed [this](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#exporter-selection) spec general guideline?

Reply:
I would expect the support of configuration via en variables using `otlp` would be decoupled from the name of the entrypoint. it could be that the name of the entrypoint itself should contain the signal instead of just `opentelemetry_exporter` there could be a signal specific entrypoint i.e `opentelemetry_metrics_exporter`

Reply:
Discussed on 16thSep SIG Meeting. We agreed to introduce signal specific entrypoint.

Reply:
And we'll keep the existing one for backward compat or was it decided to break this?

Reply:
@ocelotl had same thought if this was breaking change? we talked about it and decided this isn't. How do you think this comes under a breaking change?

Reply:
Two ways:

1. There could be 3rd party exporters out there that will need to replace `opentelemetry_exporter` with `opentelemetry_exporter_span` or whatever new entry point name we come up with.
2. Such exporters and the core/contrib exporters with version 1.5 and older will not working with opentelemetry-instrumentation version v0.25.0 and newer

Reply:
third party exporters case wasn't brought up and we certainly don't want to break.

Reply:
I still think this is a breaking change but it is a breaking change in the `opentelemetry-instrument` command which comes from `opentelemetry-instrumentation` package which is not 1.0 yet so I guess we can break it.